docs/metrics.md

```diff
@@ -7,22 +7,31 @@ Log metrics collecting metrics related to ZIO logging (all `ZIO.log*` functions)
 As ZIO core supporting multiple loggers, this logging metrics collector is implemented as specific `ZLogger` 
 which is responsible just for collecting metrics of all logs - `ZIO.log*` functions.
 
-Metrics layer:
+The Metrics layer
 
 ```scala
 val layer = zio.logging.logMetrics
 ```
 
+will add a default metric named `zio_logger_total` with the label `level` which will be 
+incremented for each log message with the value of `level` being the corresponding log level in lower case.
+
 Metrics:
-* zio_logger_all_total - logs count for `LogLevel.All`
-* zio_logger_fatal_total - logs count for `LogLevel.Fatal`
-* zio_logger_error_total - logs count for `LogLevel.Error`
-* zio_logger_warn_total - logs count for `LogLevel.Warning`
-* zio_logger_info_total - logs count for `LogLevel.Info`
-* zio_logger_debug_total - logs count for `LogLevel.Debug`
-* zio_logger_trace_total - logs count for `LogLevel.Trace`
-* zio_logger_none_total - logs count for `LogLevel.None`
+* `LogLevel.All` -> `all`
+* `LogLevel.Fatal` -> `fatal`
+* `LogLevel.Error` -> `error`
+* `LogLevel.Warning` -> `warning`
+* `LogLevel.Info` -> `info`
+* `LogLevel.Debug` -> `debug`
+* `LogLevel.Trace` -> `trace`
+* `LogLevel.None` -> `none`
```

Comment:
label for LogLevel.Warning is OFF
then this should be:

`LogLevel.None` -> `off`

NOTE: here is bug also in original documentation